src/endpoints/account/list_accounts.rs

```diff
@@ -0,0 +1,30 @@
+use super::Account;
+
+use crate::framework::endpoint::{Endpoint, Method};
+
+/// List Accounts
+/// List all accounts you have ownership or verified access to
+/// https://api.cloudflare.com/#accounts-list-accounts
+pub struct ListAccounts {
+    pub params: ListAccountsParams,
+}
+
+impl Endpoint<Vec<Account>, ListAccountsParams> for ListAccounts {
+    fn method(&self) -> Method {
+        Method::Get
+    }
+    fn path(&self) -> String {
+        format!("accounts")
+    }
+    fn query(&self) -> Option<ListAccountsParams> {
+        Some(self.params.clone())
+    }
+}
+
+#[serde_with::skip_serializing_none]
+#[derive(Serialize, Clone, Debug, Default)]
+pub struct ListAccountsParams {
+    pub page: Option<u32>,
+    pub per_page: Option<u32>,
+    pub direction: Option<String>
```

Comment:
i think documenting with a comment that this will only work with the strings `asc` and `desc` is worthwhile.

Reply:
i'm gonna do the enum haha